src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs

```diff
@@ -140,5 +140,80 @@ public bool UpdateTag(string faceInfoId, string tag)
                 Interop.Face.Destroy(handle);
             }
         }
+
+        /// <summary>
+        /// Inserts a new face information to the database with the specified media ID, area, orientation.
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/content.write</privilege>
+        /// <param name="mediaId">The media ID to be associated with.</param>
+        /// <param name="area">The region of face in the media.</param>
+        /// <param name="orientation">The orientation of specified media.</param>
+        /// <returns>The <see cref="FaceInfo"/> containing the results.</returns>
+        /// <exception cref="InvalidOperationException">The <see cref="MediaDatabase"/> is disconnected.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaDatabase"/> has already been disposed of.</exception>
```

Comment:
The <see cref="MediaDatabase"/> has already been disposed of what?

Suggestion:
  /// <exception cref="ObjectDisposedException">The <see cref="MediaDatabase"/> has already been disposed. </exception>